[TASK] List common configuration file locations as files (#5283)
TYPO3-Documentation/TYPO3CMS-Reference-CoreApi: Documentation/ExtensionArchitecture/FileStructure/Configuration/RequestMiddlewaresPhp.rst

:file:`RequestMiddlewares.php`
================================

..  typo3:file:: RequestMiddlewares.php
    :scope: extension
    :path: /Configuration/
    :regex: /^.*Configuration\/RequestMiddlewares\.php$/
    :shortDescription: Configuration of user-defined middlewares for frontend and backend

>     Configuration of user-defined middlewares for frontend and backend. Extensions
>     that add middlewares or disable existing middlewares configure them in this
>     file. The file must return an array with the configuration.

Reviewer: ```suggestion
    Configuration of custom middleware for frontend and backend. Extensions that
    add middleware or disable existing middleware are configured in this file. The
    file must return an array with the configuration.
```

    See :ref:`Configuring middlewares <request-handling-configuring-middlewares>`
    for details.

> .. include:: /CodeSnippets/Manual/Extension/Configuration/RequestMiddlewares.rst.txt

Reviewer: ```suggestion
..  include:: /CodeSnippets/Manual/Extension/Configuration/RequestMiddlewares.rst.txt
```
